content/admin/onboard_team/CreateUsers.md

```diff
@@ -4,41 +4,106 @@ description: "Creating Users in Contrast TeamServer"
 tags: "Admin onboarding TeamServer user settings license defend protection create"
 -->
 
-Users can be created by the System Administrator, as well as by the Organization Administrator. All users created in Contrast will be required to have a default organization and a default role within that organization. Users currently need to be added to Contrast one user at a time unless integrated with [Active Directory](installation-setupauth.html#ad) or [LDAP](installation-setupauth.html#ldap).
+System and Organization Administrators can create users individually, in groups, or through [Active Directory](installation-setupauth.html#ad) or [LDAP](installation-setupauth.html#ldap) integrations. All users are required to have a default organization and a default role within that organization. 
 
-## Creating Users as a System Administrator
+>**Note:** Verify the [Roles](admin-manageorgsroleperm.html#roles) that you want to assign to users so that they have the appropriate level of functionality.
 
-* Log in to Contrast 
-* Navigate to the user menu in the upper right 
-* Select **SuperAdmin** in the "Use Contrast Security as:" section
-* Select **Users** from the top navigation
-* Click the **Add User** button
-* Provide the required inputs (Email, First & Last Name, default Organization and Organization Role) 
+You may decide to designate an [Application Access Group](admin-onboardteam.html#group) which can provide more administrative function for an application to the users, or restrict what applications the users in the group can view or modify.
```

Comment:
Suggestion: "...which can provide more administrative application functionality to the users, or restrict which applications those users can view or modify."